packages/patternfly-4/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -25,7 +26,7 @@ export interface NavigationProps extends React.HTMLProps<HTMLElement> {
   /** Accessible label for the pagination component */
   paginationTitle?: string;
   /** The number of the current page */
-  page: number;
+  page: number | React.ReactText;
```

Comment:
`type ReactText = string | number;`, so can we just do `string | number` or just `React.ReactText`?